README.rst

```diff
@@ -347,10 +347,11 @@ Release Notes
 User Facing
 ~~~~~~~~~~~
 
-- Removed the partial implementation of the Mesh shape. [PR `#172`_]
-- Overhauled the code generation package to make the generators more easily
+- Support Python 3.3 and 3.4. [PR `#38`_]
```

Comment:
If you want to have present tense, that's fine but then they all should be changed. Maybe revert, except for yours and then make a new PR with this and also add a note in `CONTRIBUTING.rst` that says how the release notes should be formatted.